src/ape/managers/compilers.py

```diff
@@ -116,6 +116,48 @@ def compile(self, contract_filepaths: List[Path]) -> Dict[str, ContractType]:
 
         return contract_types_dict  # type: ignore
 
+    def fetch_imports(
+        self, contract_filepaths: List[Path], base_path: Optional[Path]
+    ) -> Dict[str, List[str]]:
+        """
+        Combines import dicts from all compilers, where the key is a contract path
+        and the value is a list of import paths.
+
+        Args:
+            contract_filepaths (List[pathlib.Path]): A list of source file paths to compile.
+            base_path (Optional[pathlib.Path]): Optionally provide the base path, such as the
+              project ``contracts/`` directory. Defaults to ``None``. When using in a project
+              via ``ape compile``, gets set to the project's ``contracts/`` directory.
+
+        Returns:
+            Dict[str, List[str]]
+        """
+        imports_dict: Dict[str, List[str]] = {}
+
+        for _, compiler in self.compiler_manager.registered_compilers.items():
```

Comment:
Surprised this doesn't segfault
```suggestion
        for _, compiler in self.registered_compilers.items():
```

Reply:
Updated [850e1e3](https://github.com/ApeWorX/ape/pull/730/commits/850e1e3efcea6fa2f036a5935cfd87eb8ddfb9e0)